neo/Ledger/MemoryPool.cs

```diff
@@ -489,5 +489,42 @@ internal bool ReVerifyTopUnverifiedTransactionsIfNeeded(int maxToVerify, Snapsho
 
             return _unverifiedTransactions.Count > 0;
         }
+
+        /// <summary>
+        /// This function return statistics about the best _maxTxPerBlock and _maxLowPriorityTxPerBlock txs of the mempool
+        /// The set of TXs used for calculating the metrics are taken for the ones most prone to be added to the next block
+        /// It is noteworthy that this call may vary from node to node, since mempool is in constantly updating
+        /// 
+        /// Note: SimplePolicy (IPolicyPlugin) is needed for loading _maxTxPerBlock and _maxLowPriorityTxPerBlock values correctly
+        /// </summary>
+        /// <returns> Return a list of Fixed8 values with current statistics of : 
+        /// [sortedHP_avgNetFees,sortedLP_avgNetFees,unverifiedHP_avgNetFees,unverifiedLP_avgNetFees,
+        /// sortedHP_avgNetFeesPerByte,sortedLP_avgNetFeesPerByte,unverifiedHP_avgNetFeesPerByte,unverifiedLP_avgNetFeesPerByte]
+        /// </returns>
+        public List<Fixed8> GetStatisticsForNextBlock()
```

Comment:
Instead of returning a list, an object with named fields would be better, and then just have a ToJson method on the object.

Reply:
Sounds good. Could please make this change?